cas-server-core/src/main/java/org/jasig/cas/util/PrivateKeyFactoryBean.java

```diff
@@ -44,17 +44,13 @@
     private String algorithm;
 
     @Override
-    protected Object createInstance() throws Exception {
-        final InputStream privKey = this.location.getInputStream();
-        try {
+    protected PrivateKey createInstance() throws Exception {
+        try (final InputStream privKey = this.location.getInputStream()) {

```

Comment:
Nice little improvement. I hope we'll begin using try-with-resources more broadly.